build.gradle

```diff
@@ -220,6 +226,18 @@ subprojects {
   tasks.withType(ScalaCompile) {
     scalaCompileOptions.useAnt = false
 
+    scalaCompileOptions.additionalParameters = [
+      "-deprecation",
+      "-unchecked",
+      "-encoding", "utf8",
+      "-target:jvm-1.7",
+      "-Xlog-reflective-calls",
+      "-feature",
+      "-language:postfixOps",
+      "-language:implicitConversions",
```

Comment:
Do we want to globally enable implicit conversions? Since we don't use them a lot (I think) and we are favoring JavaConverters vs JavaConversions we could just use `import scala.language.implicitConversions` where necessary.